Fix issues with IdnaConvert class

mallardduck opened the pull request:
This PR should fix issues related to ef87bf254a746544430f001d4b25e1e76e8c5ab8

mallardduck committed:
Fix issues with IdnaConvert class

gdevlugt reviewed (approved):
This fixes this issue for me (using dev-master and PHP 7.0.8)

mallardduck commented:
@abcdmitry, hey man I was wondering if we can get this one merged in?

mallardduck committed:
Fix GoDaddy parsing

mallardduck committed:
PSR2 all the code

...well except namespacing it all.

mallardduck committed:
Fix error casued by whois lookup of IP

mallardduck committed:
Fix ip lookup

mallardduck committed:
Fix $r var type

Unsure how or why this worked when $r was being defined as a string....but not it's not working.

(likely something PHP7.1 related)

Setting $r to be an empty array here fixes it for me.

mallardduck committed:
Update for PHP 7.2

NikoGrano commented:
Hi!

If you see this issue still relevant and could be fixed by this PR. Please submit it to https://github.com/niko9911/phpWhois.

I will make sure it will be checked and merged if necessary. Thank you for contributing.

mallardduck commented:
Thanks @Niko9911  - I actually wrote my own though in the end. My real needs were much more simple than this library, so rather than forking it I just made a thin RFC3912 client. [mallardduck/php-whois-client](https://github.com/mallardduck/php-whois-client)
